web-ng/src/app/components/feature-panel/feature-panel.component.html

```diff
@@ -28,11 +28,21 @@
       </mat-card-subtitle>
     </mat-card-header>
     <mat-card-content>
-      <div>
         <div *ngFor="let field_entry of observation.form.fields ">
-          <label class="field-label">{{ field_entry[1].label.get(lang) }}</label>
-          <div class="field-response">{{ observation.responses?.get(field_entry[1].id)?.value }}</div>
-        </div>
+          <div *ngIf="field_entry[1].type == fieldTypes.TEXT">
+            <label class="field-label">{{ field_entry[1].label.get(lang) }}</label>
```

Comment:
This might throw error if label is undefined, cannot read property get

Reply:
good point^^